packages/Search/src/details/DTK_DetailsTreeConstruction_def.hpp

```diff
@@ -17,7 +17,8 @@
 #include <DTK_DBC.hpp>
 #include <DTK_DetailsAlgorithms.hpp>
 #include <DTK_DetailsMortonCode.hpp> // morton3D
-#include <DTK_DetailsUtils.hpp>      // iota
+#include <DTK_DetailsNode.hpp>
```

Comment:
Did you run "include-what-you-use"?  How did you determine this include was missing?  Just curious.

Reply:
No. It just did not compile.